src/tools/Brush.js

```diff
@@ -53,7 +53,7 @@ const _Tool = types
     group: 'segmentation',
     shortcut: 'B',
     smart: true,
-    isDrawingTool: true,
+    allowRegionUnselectionOnToolChange: false,
```

Comment:
we have a lot of users with different behaviours, so they may get used to this unselection or even expect it, so it's better to add FF here, so with FF off it will remain the same.